Source/ACE/Entity/Door.cs

```diff
@@ -104,22 +105,25 @@ public float ResetInterval
             private set;
         }
 
+        private double? resetTimestamp;
         public double? ResetTimestamp
         {
-            get;
-            private set;
+            get { return resetTimestamp; }
```

Comment:
fields are public so that you can read them for debugging in game via assessment.